docs/extending/plugins-list.md

```diff
@@ -140,3 +140,7 @@
   - `verifyConditions`: Validate configuration and verify ```HEROKU_API_KEY```
   - `prepare`: Update the package.json version and create release tarball
   - `publish`: Publish version to heroku
+- [semantic-release-mattermost](https://github.com/ttrobisch/semantic-release-mattermost)
+  - `verifyConditions`: Verify that the webhook is setup and release-notes-generator is present.
+  - `success`: Send a message about the new release and its notes to a mattermost webhook.
```

Comment:
can you link `mattermost` to a place where people can find out what it is?

Reply:
good idea, sure